app/views/dashboard/protocols/form/study_form_sections/_study_information.html.haml

```diff
@@ -30,10 +30,12 @@
         = form.label :research_master_id, class: 'col-lg-2 control-label rm-id' do
           = link_to t(:protocols)[:studies][:information][:research_master_id], RESEARCH_MASTER_LINK, target: '_blank'
         .col-lg-3
-          = form.text_field :research_master_id, class: 'form-control research-master-field', readonly: action_name == "edit" && admin
-        - if action_name == "edit" && admin
+          = form.text_field :research_master_id, class: 'form-control research-master-field', readonly: action_name == "edit" && admin, disabled: rmid_server_down
```

Comment:
As discussed with @jwiel86 on Slack, there is a bug where if you press "Save" on a new Protocol when there are errors and when the server is down, this field is no longer disabled.

Reply:
@kyle-glick This bug has now been fixed, see app/views/protocols/create.js.coffee and app/views/dashboard/protocols/create.js.coffee.  Thank you for manually testing!